editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -278,11 +278,15 @@ export const editorStore: StateCreator<
           ) {
             id
             slug
-            updated_at
-            operations(limit: 1, order_by: { id: desc }) {
+            updatedAt: updated_at
```

Comment:
Note that `GetFlow` is still ordered by `flows.updated_at`, and can't actually be ordered easily at the GraphQL level by `flows.operations.created_at` - so this is going to appear a bit strange on production! 

Do we want to keep the `updated_at` ordering, switch to alphabetical flow order, or something else (eg "online" flows at top, "offline" flows at bottom)?